NorthstarDLL/core/hooks.cpp

```diff
@@ -9,6 +9,10 @@
 #include <filesystem>
 #include <Psapi.h>
 
+#define IGO_DLL L"IGO64.dll"
+
+bool disable_igo = false;
```

Comment:
Might be better to just use a static bool inside the function scope for this?
also camelCase please :(